fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/RegistryRemapCallback.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.event.registry;
+
+import it.unimi.dsi.fastutil.ints.Int2IntMap;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.impl.registry.ListenableRegistry;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * The remapping process functions as follows:
+ *
+ * - RegistryRemoveObjectCallbacks are called to remove any objects culled in the process, with the old numeric ID.
+ * - RegistryRemapCallback is emitted to allow remapping the IDs of objects still present.
+ * - RegistryAddObjectCallbacks are called to add any objects added in the process, with the new numeric ID.
```

Comment:
RegistryRemoveEntryCallback/RegistryAddEntryCallback, right?

Reply:
Yes, I missed the rename